Add documentation for (alpha) feature ServiceIPStaticSubrange (#32345)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates.md

  topology of the cluster. See
  [ServiceTopology](/docs/concepts/services-networking/service-topology/)
  for more details.

> - `ServiceIPStaticSubrange`: Enable a new strategy for Services ClusterIP allocations, so users can

Reviewer: Nit + adding on to Tim's suggestions above.
```suggestion
- `ServiceIPStaticSubrange`: Enables a strategy for Services ClusterIP allocations, whereby users can
```

  ClusterIP range is subdivided. Dynamic allocated ClusterIP addresses will be allocated preferently
  from the upper range allowing users to assign static ClusterIPs from the lower range with a low
  risk of collision. See

>   [ServiceIPStaticSubrange](/docs/concepts/services-networking/service.md#avoiding-collisions)

Reviewer 2: ```suggestion
  [ServiceIPStaticSubrange](/docs/concepts/services-networking/service/avoiding-collisions)
```

Author: TIL, can you reference sections this way?

Reviewer 2: The md files you see in the git repo will be converted into HTML on a page-by-page basis, so no ".md" would be needed. But my suggestion isn't correct either. It should be:

```
  [ServiceIPStaticSubrange](/docs/concepts/services-networking/service/#avoiding-collisions)
```

Reviewer 3: The text of the hyperlink should be “Avoiding collisions”.

>   for more details.

Reviewer 3: We don't ask readers to refer to KEPs to understand Kubernetes. Please instead add a document that describes the feature into the website repo.

If you're looking for a place to hang that new documentation off, it may help to move PR https://github.com/kubernetes/website/pull/30817 forward first - but you don't have to.

Author: Yeah, we should do #30817 , but I'll not mix it with this, the scope for #30817 is too broad and there are certain concepts that doesn't apply anymore, I'll comment there

  topology of the cluster. See
  [ServiceTopology](/docs/concepts/services-networking/service-topology/)
  for more details.

> - `ServiceIPStaticSubrange`: Enable a strategy for Services ClusterIP allocations, whereby users can

Reviewer: ```suggestion
- `ServiceIPStaticSubrange`: Enables a strategy for Services ClusterIP allocations, whereby users can
```
